### PR TITLE
update evangelists.yaml

### PR DESCRIPTION
**Pull request by egodoy-cl**
- name: Eduardo Godoy
  email: egodoy@agc.cl
  twitter: EdoGodoy
  linkedin: https://www.linkedin.com/in/EduardoGodoy/
  locations: Santiago, Chile
  language: Spanish, English
  has_talks: Y
  has_training: Y
  notes: Virtual CISO, Security Advisor

**- Summary**

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: site/data/evangelists.yaml [hunk 1]
```diff
@@ -550,3 +550,12 @@
```
Comment (kingthorin) [inline site/data/evangelists.yaml:552]:
```suggestion

```

It would be nice if there was just one blank between, as with the others.

**Review by kingthorin — CHANGES_REQUESTED** [review]

**Comment by thc202** [comment]
This now has conflicts.

**Comment by thc202** [comment]
@egodoy-cl do you agree with DCO? We can address the remaining tasks.

**Comment by kingthorin** [comment]
@egodoy-cl do you plan to address the earlier comments/questions, and now conflicts?

**Comment by kingthorin** [comment]
Closing. It’s been over a month.

**Pull request closed without merging**